lib/src/components/NxFilterInput/__tests__/NxFilterInput.test.tsx

```diff
@@ -5,160 +5,108 @@
-    const component = shallowComponent({
+  it('does not pass validatable, validationErrors, or type props to the Input', function() {
+    const input = quickRender({
       validatable: true,
       validationErrors: 'It\'s all wrong',
       type: 'textarea'
-    } as Partial<Props>);
-
-    expect(component).not.toHaveProp('validatable');
-    expect(component).not.toHaveProp('validationErrors');
-    expect(component).not.toHaveProp('type');
-  });
+    } as Partial<Props>).getByRole('textbox');
 
-  it('passes isPristine = false to the PrivateNxTextInput', function() {
-    expect(shallowComponent()).toHaveProp('isPristine', false);
+    expect(input).not.toHaveAttribute('validatable', true);
+    expect(input).not.toHaveAttribute('validationErrors', 'It\'s all wrong');
+    expect(input).not.toHaveAttribute('type', 'textarea');
   });
 
-  it('puts the ref on the PrivateNxTextInput', function() {
+  it('sets ref on the Input', function() {
     const ref = React.createRef<HTMLDivElement>(),
+        el = renderEl({ ref } as Partial<Props>);
 
-        // note: the fragment is necessary to get around an enzyme issue:
-        // https://github.com/enzymejs/enzyme/issues/1852#issuecomment-433145879
-        component = mount(<><NxFilterInput { ...minimalProps } ref={ref} /></>),
-        domNode = component.find('div.nx-filter-input').getDOMNode();
-
-    expect(ref.current).toBe(domNode);
+    expect(ref.current).toBe(el);
   });
 
-  it('passes an faFilter icon with nx-icon--filter-icons as the prefixContent when searchIcon is undefined',
-      function() {
-        const IconFixture = function() {
-              return shallowComponent().prop('prefixContent');
-            },
-            icon = shallow(<IconFixture />);
-
-        expect(icon).toMatchSelector(NxFontAwesomeIcon);
-        expect(icon).toHaveProp(icon, faFilter);
-        expect(icon).toHaveClassName('nx-icon--filter-icons');
-      }
-  );
-
-  it('passes an faSearch icon with nx-icon--filter-icons as the prefixContent when searchIcon is true', function() {
-    const IconFixture = function() {
-          return shallowComponent({ searchIcon: true }).prop('prefixContent');
-        },
-        icon = shallow(<IconFixture />);
-
-    expect(icon).toMatchSelector(NxFontAwesomeIcon);
-    expect(icon).toHaveProp(icon, faSearch);
-    expect(icon).toHaveClassName('nx-icon--filter-icons');
+  it('renders a filter icon when searchIcon is undefined', function() {
+    const icons = quickRender().getAllByRole('img', { hidden: true });
+
+    expect(icons[0]).toHaveAttribute('data-icon', 'filter');
   });
 
-  it('passes an NxButton with the Close icon and nx-btn--clear classname as the suffixContent', function() {
-    const BtnFixture = function() {
-          return shallowComponent().prop('suffixContent');
-        },
-        btn = shallow(<BtnFixture />);
+  it('renders a search icon when searchIcon is true', function() {
```

Comment:
As we convert to RTL some things that were unit tests will need to become visual tests.  Choice of icon is one of those things.  I think these two should be removed.  The corresponding visual tests already exist.

Reply:
Removed here 022713cb070daad192ed4d837776f74f260c2fef.